build.bash

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eo pipefail
+
+mkdir -p ./build
+pushd ./build
+
+cmake .. -G Ninja
+ninja
```

Comment:
There is probably a much cleaner cmake invocation to do this all in one command but I didn't bother there...